conda_forge_tick/migrators/version.py

```diff
@@ -622,6 +632,22 @@ def pr_body(self, feedstock_ctx: FeedstockContext) -> str:
             )
         for p in pred:
             body += template.format(name=p[0], new_version=p[1])
+
+        update_deps = feedstock_ctx.attrs.get("conda-forge.yml", {}).get("bot", {}).get("update_deps", False)
```

Comment:
We should always hint.

Reply:
Maybe, do you want to hint if we update directly?

Reply:
yes. we should say we did it in the pr text